telegram/chatmember.py

```diff
@@ -29,7 +29,15 @@
 
 
 class ChatMember(TelegramObject):
-    """This object contains information about one member of a chat.
+    """Base class for Telegram ChatMember Objects..
```

Comment:
```suggestion
    """Base class for Telegram ChatMember Objects.
```
Had to manually type this out since github removed suggestions?

Reply:
> Had to manually type this out since github removed suggestions?

works fine for me …

Reply:
ah turns out the suggestion button doesn't show up if you view changes from one commit. Which makes sense, but should show up for the latest commit ig